.gitignore

```diff
@@ -6,3 +6,4 @@ yarn-debug.log*
 yarn-error.log*
 test/unit/coverage
 package-lock.json
+.idea
```

Comment:
and no need to add .idea folder to gitignore, because not all programmers use it, so please remove this commit
FYI, you can create global gitgitnore for your laptop
https://help.github.com/articles/ignoring-files/

Reply:
I reverted the commit. I was wondering how much of importance is not adding extra line in a .gitignore file? Since jetbrains has very large user base